client/scss/layouts/_add-multiple.scss

```diff
@@ -36,7 +36,7 @@
     padding: 1em;
   }
 
-  .left {
+  .w-header__primary {
```

Comment:
I do not think this ever gets used in a header component - please can you double check where these styles are used, this looks like it should not be changed.